src/mon/MgrMap.h

```diff
@@ -36,17 +38,21 @@ class StandbyInfo
 
   void encode(bufferlist& bl) const
   {
-    ENCODE_START(1, 1, bl);
+    ENCODE_START(2, 1, bl);
```

Comment:
Is this inconsistent with the HEAD_VERSION in MMgrBeacon.h? Or do I just not understand this piece?